docs/in-app-messaging/usage/index.md

```diff
@@ -49,6 +49,8 @@ Firebase Console are automatically handled and displayed on your user's device.
 
 This module provides a JavaScript API to allow greater control of the displaying of these messages.
 
+# Limitations
+Accorrding to [Gihthub isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch. This limit is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (for example: Analytics event or programmatically triggered in-app-messaging campaigns), there is no daily rate limit.
```

Comment:
```suggestion
According to [Github isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch. This limit is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (for example: Analytics event or programmatically triggered in-app-messaging campaigns), there is no daily rate limit.
```